backend/test/services.members.spec.js

```diff
@@ -403,11 +431,50 @@ describe('services', function () {
       })
 
       describe('#updateServiceAccount ', function () {
-        it('should not delete a serviceaccount from a different namespace', async function () {
+        it('should not update a serviceaccount from a different namespace', async function () {
+          const id = 'system:serviceaccount:garden-foreign:robot-foreign-namespace'
+          const item = memberManager.subjectList.get(id)
+          try {
+            await memberManager.updateServiceAccount(item, {})
+            expect.fail('should throw an error')
+          } catch (err) {
+            expect(err).to.be.instanceof(UnprocessableEntity)
```

Comment:
We have many UnprocessableEntity errors check significant part of message with regex